doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -642,6 +642,31 @@ The `mz_sessions` table contains a row for each active session in the system.
 | `client_ip`      | [`text`]                       | The IP address of the client that initiated the session.                                                                  |
 | `connected_at`   | [`timestamp with time zone`]   | The time at which the session connected to the system.                                                                    |
 
+
+## `mz_network_policies`
+The `mz_network_policies` table contains a row for each network policy.
+
+<!-- RELATION_SPEC mz_internal.mz_network_policies -->
+| Field            | Type                  | Meaning                                                                                                         |
+| -----------------| ----------------------| --------                                                                                                        |
+| `id`             | [`text`]              | The ID of the network policy.                                                                                   |
+| `name`           | [`text`]              | The name of the network policy.                                                                                 |
+| `owner_id`       | [`text`]              | The role ID of the owner of the network policy. Corresponds to [`mz_catalog.mz_roles`](../mz_catalog#mz_roles). |
+| `privileges`     | [`mz_aclitem array`]  | The privileges belonging to the network policy.                                                                 |
+| `oid`            | [`oid`]               | A [PostgreSQL-compatible OID][`oid`] for the network policy.                                                    |
+
+## `mz_network_policy_rules`
+The `mz_network_policy_rules` table contains a row for each network policy rule.
+
+<!-- RELATION_SPEC mz_internal.mz_network_policy_rules -->
+| Field            | Type       | Meaning                                                                                                |
+| -----------------| ----------------------| --------                                                                                    |
+| `name`           | [`text`]   | The name of the network policy rule. Can be used with `network_policy_id` to form a unique identifier. |
+| `policy_id`      | [`text`]   | The id the network policy the rule is part of. Can be used with `name` to form a unique identifier.    |
```

Comment:
even smaller nit (feel free to ignore): maybe "The identifier of the network policy associated with the rule." ?

Reply:
The rule is really an attribute (one of a set of rules) of a policy so I was a bit hesitant to use the world "associated" as it might imply you could associate the rule with another policy, which is not possible. 

what I current have is a bit clunky though.

Reply:
👍 maybe "The ID of the network policy to which the rule belongs"?  (but, that might be overly "4th grade english grammar"-ish.  What you have is fine as well as my comment is squarely in the ignorable category).